wasmpay-ui/src/features/dashboard/components/dashboard.tsx

```diff
@@ -3,19 +3,22 @@ import {AccountBalance} from './account-balance';
 import {Transactions} from '@/features/transactions/components/transactions';
 import {Chat} from '@/features/chat/components/chat';
 import {TransactionModal} from '@/features/transactions/components/transaction-modal';
+import {useTransactions} from '@repo/common/hooks/useTransactions';
 
 function Dashboard(): React.ReactElement {
+  const {transactions, balance, isLoading, reloadTransactions} = useTransactions();
+
```

Comment:
@lachieh I think this is the source of this issue
<img width="713" alt="image" src="https://github.com/user-attachments/assets/0459116e-4698-487b-9ab5-df3b3b7be700" />